modules/typelevelShell.nix

```diff
@@ -60,24 +60,13 @@ in
         ] ++ optionals cfg.nodejs.enable [
           pkgs.nodejs
           pkgs.yarn
-        ] ++ optionals cfg.native.enable [
-          pkgs.stdenv
-          pkgs.boehmgc
-          pkgs.libunwind
-          pkgs.clang
-          pkgs.zlib
         ];
 
         env = [
           {
             name = "JAVA_HOME";
             value = "${cfg.jdk.package}";
           }
-        ] ++ optionals cfg.native.enable [
-          {
-            name = "LLVM_BIN";
-            value = "${pkgs.clang}/bin";
-          }
```

Comment:
I don't see where anything uses this variable.

Reply:
It's potentially used by Scala Native itself to discover clang and friends.
https://github.com/scala-native/scala-native/blob/3498dbfb929d22b300c65fdc78fd0e12aacf349b/tools/src/main/scala/scala/scalanative/build/Discover.scala#L31-L43

Reply:
Eh, doesn't hurt to keep it, I guess.